app/assets/stylesheets/components/navbar.scss

```diff
@@ -183,70 +183,78 @@
         display: none;
       }
     }
-    .background {
-      background-color: #fff;
-      -webkit-box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.05);
-      -moz-box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.05);
-      box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.05);
-      position: absolute;
-      top: 0;
-      left: 0;
+  }
+  .modal-fullscreen-menu {
```

Comment:
The nesting of classes under this is getting a bit dangerous, I think this belongs in its own `modal_fullscreen_menu.scss` file so its more of a component and isn't as scoped.